integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/search/LuceneMatchSearchPredicateIT.java

```diff
@@ -66,21 +69,51 @@ public void match_skipAnalysis_normalizedStringField() {
 		assertThat( query ).hasNoHits();
 	}
 
+	@Test
+	public void searchAnalyzer_normalizedStringField() {
+		StubMappingScope scope = indexManager.createScope();
+
+		SearchQuery<DocumentReference> query = scope.query()
+				// the matching parameter will be tokenized even if the field has a normalizer
+				.predicate( f -> f.match().field( "normalizedSearchAnalyzedField" ).matching( "Auster Coe" ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, "2", "3" );
+	}
+
 	private void initData() {
 		IndexIndexingPlan<? extends DocumentElement> plan = indexManager.createIndexingPlan();
-		plan.add( referenceProvider( "1" ), document -> document.addValue( indexMapping.normalizedStringField, TEST_TERM ) );
+		plan.add( referenceProvider( "1" ), document -> {
+			document.addValue( indexMapping.normalizedStringField, TEST_TERM );
+			document.addValue( indexMapping.normalizedSearchAnalyzedField, "Irving" );
+		} );
+		plan.add( referenceProvider( "2" ), document -> {
+			document.addValue( indexMapping.normalizedSearchAnalyzedField, "Auster" );
+		} );
+		plan.add( referenceProvider( "3" ), document -> {
+			document.addValue( indexMapping.normalizedSearchAnalyzedField, "Coe" );
+		} );
 		plan.execute().join();
 	}
 
 	private static class IndexMapping {
 		final IndexFieldReference<String> normalizedStringField;
+		final IndexFieldReference<String> normalizedSearchAnalyzedField;
 
 		IndexMapping(IndexSchemaElement root) {
 			normalizedStringField = root.field(
 					"normalizedStringField",
 					c -> c.asString().normalizer( DefaultAnalysisDefinitions.NORMALIZER_LOWERCASE.name )
 			)
 					.toReference();
+
+			normalizedSearchAnalyzedField = root.field(
+					"normalizedSearchAnalyzedField",
+					c -> c.asString().normalizer( DefaultAnalysisDefinitions.NORMALIZER_LOWERCASE.name )
+							.searchAnalyzer( OverrideAnalysisDefinitions.ANALYZER_WHITESPACE_LOWERCASE.name )
```

Comment:
If you want to allow a search analyzer overriding a normalizer, you should also:

* add the option to `@KeywordField`
* test it
* update the documentation of the `searchAnalyzer` method in `StringIndexFieldTypeOptionsStep` (it only mentions overriding the analyzer).

Reply:
I added the test to probe the backends.
So it seems that the Lucene backend potentially can support it, in contrast Elasticsearch not.

I we added this option to `@KeywordField`, shouldn't it be supported by both backends?

If you want we can talk on zulip.

Reply:
I'll drop this commit, as we decided on zulip